activedoptree.py

```diff
@@ -158,8 +206,8 @@ def _ptree_to_cgel(self) -> ParentedTree:
 		Assumes that tree has been processed by apply_standard_labels().
 		Outputs both a CGELTree object and a ParentedTree object, the latter of which is the cleaned-up version of the input tree with a canonicalized position for punctuation preterminals/terminals.
```

Comment:
I see that this function returns a ParentedTree. Where does it output a CGELTree object? I'm confused about the name `_ptree_to_cgel` if the output is a ParentedTree.

Reply:
Whoops, both the docstring and the type hint need to be updated here. The function outputs just a CGELTree object.

Reply:
Looking at it some more, I am wondering if the return type annotation is wrong and should be `CGELTree`. I think it is stripping punctuation, converting to a ParentedTree, and then to a CGELTree. I wonder if there is some redundancy with the cgel2export endpoint in app.py.